AutorunsToWinEventLog/AutorunsToWinEventLog.ps1

```diff
@@ -32,3 +32,44 @@ Foreach ($item in $autorunsArray) {
   $item = $(Write-Output $item  | Out-String -Width 1000)
   Write-EventLog -LogName Autoruns -Source AutorunsToWinEventLog -EntryType Information -EventId 1 -Message $item
 }
+
```

Comment:
Can you create a comment block on line 6 similar to this PR: https://github.com/palantir/windows-event-forwarding/pull/23/files#diff-4bda8306d7651cf46ff5b669ea0838c4R8

It should read:
```
# The following event IDs are in use:
# 1 - Sysinternals Autoruns results
# 2 - Local Group Principals 
```